_data/projects/AMPProject.yml

```diff
@@ -0,0 +1,13 @@
+name: The AMP Project (Accelerated Mobile Pages)
+desc: The AMP Project provides a framework (built on standard web technologies) to make it easy to build fast web pages.
+site: https://github.com/ampproject/amphtml/
+tags:
+- amp
+- css
+- documentation
+- html
+- javascript
+- web
+upforgrabs:
+  name: Great First Issues
+  link: https://github.com/ampproject/amphtml/milestone/25
```

Comment:
OK, done. :)  It does seem like labels are a more standard way of tracking these, and it shouldn't be too hard to maintain both since the rate of GFI creation is manageable.